Makefile

```diff
@@ -8,7 +8,10 @@ OPERATOR_IMAGE     ?= kubevirt/hyperconverged-cluster-operator
 REGISTRY_NAMESPACE ?=
 
 build: $(SOURCES) ## Build binary from source
-	go build -i -ldflags="-s -w" -o _out/hyperconverged-cluster-operator ./cmd/manager
+	go build -i -ldflags="-s -w" -o _out/hyperconverged-cluster-operator ./cmd/hyperconverged-cluster-operator
```

Comment:
I think it's the operator-sdk that generated `cmd/manager/main.go`. I don't think there is any negative impact to this move but why are we moving it?

Reply:
because go install would install the binary otherwise with the name `manager`. If we move it it will give it the right name. I could probably find a workaround  to give it the right name afterwards, but this meets the normal go-expectations and makes integration into the image streams possible without any modification, and the right binary name.